src/api.rs

```diff
@@ -2813,6 +2861,18 @@ pub struct ProcessedEventUser {
     pub ip_address: Option<String>,
 }
 
+#[derive(Clone, Debug, Deserialize)]
```

Comment:
You should be able to use `#[serde(rename_all = "camelCase")]` instead I believe.

Reply:
Can you also move it near `IssueFilter` and `IssueChanges` so it's grouped together?

Reply:
Great! Thx